VisualRiders.PointOfSale.Project/Profiles/StaffMembersProfile.cs

```diff
@@ -0,0 +1,14 @@
+﻿using AutoMapper;
+using VisualRiders.PointOfSale.Project.DTOs;
+using VisualRiders.PointOfSale.Project.Models;
+
+namespace VisualRiders.PointOfSale.Project.Profiles;
+
+public class StaffMembersProfile : Profile
+{
+    public StaffMembersProfile()
+    {
+        CreateMap<CreateUpdateStaffMemberDto, StaffMember>();
```

Comment:
It's a good practice to use `MemberList.Source` when destination object has more fields than the source